server/utils/socketUsers.js

```diff
@@ -0,0 +1,18 @@
+let users = [];
```

Comment:
Using an array to store a list of online users works fine. But it would be faster to search through in a JavaScript object than in an array. This is a good optimization for performance.

Reply:
done ✔️